release/models/acl/openconfig-packet-match-types.yang

```diff
@@ -371,4 +371,28 @@ module openconfig-packet-match-types {
       decimal notation, or using a type defined by the
       ETHERTYPE identity";
   }
+
+  typedef ipv4-prefix-mask-type {
+    type union {
+      type uint8 {
+        range 0..32;
+      }
+      type oc-inet:ipv4-address;
+    }
+    description
+      "The ipv4-prefix mask to be used in acl-entries. Can be in CIDR
+      formmat or wildcard bits.";
```

Comment:
(nit) `s/formmat/format/`

Since this a `typedef` that can be used anywhere, we should probably not say where this is used. If it's single usage, prefer just using the above `type` in that place.

Reply:
Deleting this changes as this typedef is not needed anymore